### PR TITLE
airpaste: add page

### PR DESCRIPTION
**Pull request by olso**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/airpaste.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# airpaste
+
+> Share messages and files locally.
```
Comment (agnivade) [inline pages/common/airpaste.md]: "locally" might mean on the local machine. I think we should say "local network" for clarity.

File: pages/common/airpaste.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Send file:
+
+`airpaste < file.txt`
```
Comment (agnivade) [inline pages/common/airpaste.md]: Put file.txt in tokens (`{{}}`)

File: pages/common/airpaste.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+- Receive file:
+
+`airpaste > file.txt`
```
Comment (agnivade) [inline pages/common/airpaste.md]: Same here

File: pages/common/airpaste.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+- Create/Join channel:
+
+`airpaste 🏀`
```
Comment (agnivade) [inline pages/common/airpaste.md]: Is that an emoji ? Is there a way to get that from keyboard ? Or do we need to copy-paste always ?

Comment (olso) [reply pages/common/airpaste.md]: It's just an example, I thought it was fun, _airpasting a basketball_ 🤠

I could be anything e.g. `airpaste channelOne`

`Command + Control + Spacebar` on macOS for emoji input

**Comment by olso** [comment]
@agnivade I've addressed the comments :)

File: pages/common/airpaste.md (hunk 4 above)
Comment (agnivade) [reply pages/common/airpaste.md]: Let's change it to `airpaste {{channel_name}}` for clarity.

Comment (pxgamer) [reply pages/common/airpaste.md]: I'd agree with changing it to a token, emojis are often unlikely to display on Windows.

Windows Example

**Comment by pxgamer** [comment]
@olso I think this looks fine, but it may be worth changing the basketball to a token of `{{channel_name}}` for clarity as mentioned by @agnivade.

**Comment by olso** [comment]
@pxgamer @agnivade How about now? 🙂

**Comment by pxgamer** [comment]
@olso, sorry to be picky.

As per the style guide, tokens should be all lowercase, separated with underscores.

Please could you change it to `{{channel_name}}` and then it's fine by me.

**Comment by agnivade** [comment]
Made the change myself. Thanks @olso for the contribution !

**Pull request merged**